scripts/qa_crowbarsetup.sh

```diff
@@ -1265,14 +1270,16 @@ function onadmin_allocate()
     done
     echo "Sleeping 50 more seconds..."
     sleep 50
+    local controllernodes=(
+            $(get_all_discovered_nodes | head -n 2)
```

Comment:
I don't know, if we should add some condition and check how much nodes we have. For 3 nodes setup with enabled `want_sles12` and `want_ceph` SLE12 controller node will be overwritten by Storage node